Everly Precia Suresh/Arrays/delete_duplicates.cpp

```diff
@@ -0,0 +1,34 @@
+#include <iostream>
+using namespace std;
+int main(){
+    int n;
+    cout<<"Enter number of elements : ";
+    cin>>n;
+    int ar[n],freq[n];
+    cout<<"Enter Elements: ";
+
+    for(int i=0;i<n;i++){ //read
+        cin>>ar[i];
+        freq[i]=-1;
```

Comment:
What's the use of freq array?

Reply:
the freq array has count of each elements present in inputted array. for eg if ar[]={1,2,1,1,2} then freq array will contain = {3,2,0,0,0} (i.e 1=3 count , 2=2 count). The elements that contain zeros indicates that the element is already counted before.

Reply:
changed the code ! thanks for pointing this out